pkg/suggestion/NASenvelopenet/generate.py

```diff
@@ -0,0 +1,487 @@
+import sys
+import json
+import copy
+import re
+import random
+import math
+import argparse
+from collections import Counter
+
+
+class NACAlg:
+    def __init__(self, algorithm, config, stages, initcell):
+        self.algorithm =algorithm
+        self.config = config
+        print(self.config)
+        if self.algorithm == "envelopenet" or self.algorithm == "envelopenet2":
```

Comment:
Is it possible to have envelopenet2 algorithm, since you don't have it here https://github.com/kubeflow/katib/blob/976bb40b4a8f7cdbbaa8808fec60528d1750e038/pkg/suggestion/NASenvelopenet/suggestion_param.py#L4 ?

Reply:
What is the point of including deterministic or random in this suggestion?